backend/src/bundles/auth/auth.service.ts

```diff
@@ -11,9 +20,43 @@ class AuthService {
         this.userService = userService;
     }
 
+    public async signIn(
+        userRequestDto: UserSignInRequestDto,
+    ): Promise<UserSignInResponseDto> {
+        const { email, password } = userRequestDto;
+        const user = await this.userService.find(email);
+
+        if (!user) {
+            throw new HttpError({
+                message: UserValidationMessage.WRONG_CREDENTIALS,
+                status: HttpCode.BAD_REQUEST,
+            });
+        }
+
+        const { passwordHash } = user.toNewObject();
+
+        const isPwdCorrect = cryptService.compareSyncPassword(
+            password,
+            passwordHash,
+        );
+
+        if (!isPwdCorrect) {
+            throw new HttpError({
+                message: UserValidationMessage.WRONG_CREDENTIALS,
+                status: HttpCode.BAD_REQUEST,
+            });
+        }
+        const id = user.toObject().id;
+
+        const token = await createToken(id);
+
+        return { 'id': id, 'email': email, 'token': token };
```

Comment:
You can use property shorthand here:`{ id, email, token }`

Reply:
Corrected, thank you